README.md

```diff
@@ -0,0 +1,120 @@
+It has support for ACLs, sieve, LDAP, and automatic reporting to rspamd.
+Also, clustering via dsync is supported.
+
+The log is redirected to the systemd journal and the service is hardened using some systemd sandboxing.
+
+## Requirements
+
+Debian
+
+## Role Variables
+
+| Name                              | Default / Mandatory     | Description                                                                              |
+|:----------------------------------|:-----------------------:|:-----------------------------------------------------------------------------------------|
+| `dovecot_release`                 | `2.3`                   | Release of Dovecot to install (relevant for the apt repository)                          |
+| `dovecot_mail_location`           | `/var/lib/imap`         | Location for all mail-related files                                                      |
+| `dovecot_vmail_user`              | `vmail`                 | User which will own all mail                                                             |
+| `dovecot_vmail_group`             | `vmail`                 | Group which will own all mail                                                            |
+| `dovecot_dhparam`                 | :heavy_check_mark:      | Location of the TLS dhparam file                                                         |
+| `dovecot_ca_dir`                  | `/etc/ssl/certs`        | Location of the CA certificates directory                                                |
+| `dovecot_cert`                    | :heavy_check_mark:      | IMAP/managesieve certificate                                                             |
+| `dovecot_key`                     | :heavy_check_mark:      | IMAP/managesieve private key                                                             |
+| `dovecot_dsync_tls_cert`          | (:heavy_check_mark:)    | (Only if using dsync) Path to the dsync certificate                                      |
+| `dovecot_dsync_tls_key`           | (:heavy_check_mark:)    | (Only if using dsync) Path to the dsync key                                              |
+| `dovecot_listen`                  | `[*, ::]`               | IPs/ports to listen on                                                                   |
+| `dovecot_default_client_limit`    | `400`                   | Default client limit for services                                                        |
+| `dovecot_tls`                     | `required`              | Whether TLS should be offered to clients                                                 |
+| `dovecot_tls_min`                 | `TLSv1.2`               | Minimum TLS version to expect from clients                                               |
+| `dovecot_tls_ciphers`             | (see defaults/main.yml) | List of TLS ciphers to allow for users                                                   |
```

Comment:
You could link to the defaults file

Reply:
Are you sure? That makes the column so wide when viewing plain